sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/controller/AuthController.java

```diff
@@ -0,0 +1,59 @@
+package com.alibaba.csp.sentinel.dashboard.controller;
+
+import com.alibaba.csp.sentinel.dashboard.auth.AuthService;
+import com.alibaba.csp.sentinel.dashboard.auth.SimpleWebAuthServiceImpl;
+import com.alibaba.csp.sentinel.dashboard.config.DashboardConfig;
+import com.alibaba.csp.sentinel.dashboard.domain.Result;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author cdfive
+ * @since 1.6.0
+ */
+@RestController
+@RequestMapping(value = "/auth", produces = MediaType.APPLICATION_JSON_VALUE)
+public class AuthController {
+
+    private static Logger LOGGER = LoggerFactory.getLogger(AuthController.class);
+
+    @Value("${auth.username}")
+    private String authUsername;
+
+    @Value("${auth.password}")
+    private String authPassword;
+
+    @RequestMapping(value = "/login", method = RequestMethod.POST)
+    public Result login(HttpServletRequest request, String username, String password) {
+        if (StringUtils.isNotBlank(DashboardConfig.getAuthUsername())) {
+            authUsername = DashboardConfig.getAuthUsername();
+        }
+
+        if (StringUtils.isNotBlank(DashboardConfig.getAuthPassword())) {
+            authPassword = DashboardConfig.getAuthPassword();
+        }
+
+        if (!authUsername.equals(username) || !authPassword.equals(password)) {
+            LOGGER.error("login failed, error username or password. username=" + username + ", password=" + password);
+            return Result.ofFail(-1, "用户或密码错误");
```

Comment:
Maybe we need to provide English error message here, and in frontend page, we can identify the error code and parse as Chinese.

Reply:
It has been fixed.